simpleviewinc/keg-cli
# ZEN-543: Develop Tag

**Ticket**: [ZEN-543](https://jira.simpleviewtools.com/browse/ZEN-543)

## Context

* As part of ZEN-543, we determined we should change the default docker image tag from "master" to "develop" for clarity and consistency with the git branches

## Goal

* Update all references of the old "master" to "develop"
* Ensure building images uses "develop"

## Updates
* `*.config.json`
  * set the default tag in the base config
* `docs/images.md`
  * updated packages documentation to refer to "develop"
* `*.js`
  * updated references of "master" tag to "develop"

## Testing

### Setup
* Switch your local keg-cli and keg-hub branches:
  * `keg cli && keg pr 59`
  * `keg && keg pr 151`
* `keg cli update version=4.0.0` (`4.0.0` is not the version in the  package.json yet, so we have to run it manually until we make our `4.0.0` release)
* Remove all your local images (`keg d clean -f`)

### Test build
* Build these keg-hub repo images: base, tap, keg-core, keg-components, retheme
* Verify each image is tagged with "develop"
* I already pushed develop images, but feel free to push them too
